sound/soc/sof/sof-priv.h

```diff
@@ -698,4 +513,6 @@ int intel_pcm_open(struct snd_sof_dev *sdev,
 int intel_pcm_close(struct snd_sof_dev *sdev,
 		    struct snd_pcm_substream *substream);
 
+bool snd_sof_dsp_d0i3_on_suspend(struct snd_sof_dev *sdev);
```

Comment:
is this needed if you removed the helper?

Reply:
@plbossart No :)